.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "lib/cbmc"]
 	path = lib/cbmc
+	branch = develop
```

Comment:
Will this track `develop` ?

Reply:
@martin-cs Documentation to cover the submodule is in https://github.com/diffblue/gnat2goto/pull/119